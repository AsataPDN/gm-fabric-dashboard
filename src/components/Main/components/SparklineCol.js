import { spacingScale } from "style/styleFunctions";

import TableCol from "./TableCol";

const SparklineCol = TableCol.extend`
  height: ${spacingScale(3)};
  padding-top: ${spacingScale(0.5)};
  padding-bottom: ${spacingScale(0.5)};

  svg {
    height: inherit;
  }
`;

export default SparklineCol;
