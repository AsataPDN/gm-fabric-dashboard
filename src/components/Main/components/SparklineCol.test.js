import React from "react";
import { shallow } from "enzyme";
import SparklineCol from "./SparklineCol";

describe("SparklineCol", () => {
  it("matches snapshot", () => {
    const aSparklineCol = shallow(<SparklineCol />);
    expect(aSparklineCol).toMatchSnapshot();
  });
});
