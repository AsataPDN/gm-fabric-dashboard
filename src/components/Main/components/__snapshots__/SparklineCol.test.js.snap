// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SparklineCol matches snapshot 1`] = `
.c0 {
  -webkit-flex: 1 1 12%;
  -ms-flex: 1 1 12%;
  flex: 1 1 12%;
  text-align: left;
  height: 24px;
  padding-top: 4px;
  padding-bottom: 4px;
}

.c0 svg {
  height: inherit;
}

<div
  className="c0"
/>
`;
